src/localStorage/localStorage.js

```diff
@@ -0,0 +1,53 @@
+// Creating localStorage class
+
+class localStorage {
```

Comment:
where do you set objects in browsers local storage?
https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage